### PR TITLE
Itsadssd 28865 fix display of components

### PR DESCRIPTION
**Pull request by uqtgeor4**
Accordion - display fix, doc added

Card - display fix, restructure, doc added 
(Card consolidate FS, CMS style guide is out of scope for now)

### CODE REVIEW

**Review by Bcrunk — COMMENTED** [review]
I am not sure why we have 2 default examples ... it should be only one default

**Review by Bcrunk — COMMENTED** [review]
Recommend we drop the lettering of examples.  

The example names should be clear and simple

File: packages/storybook-html/src/components/accordion/accordion.docs.mdx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    <div class="accordion__content" aria-hidden="true" id="content-1" aria-labelledby="accordion-title-1">
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum auctor eros dui, vitae iaculis leo ornare eget. Etiam tristique lobortis ligula non mattis. Ut dapibus libero neque, vel tincidunt nunc tempus eget.</p>
+    </div>
+    </div>
+    ...
+    ...
+</div>
+```
+
+### b. Default using list tags
```
Comment (Bcrunk) [inline packages/storybook-html/src/components/accordion/accordion.docs.mdx:35]: Drop Default

"Using list tags" 

This is a good example of where addtional information would be helpful. 

What are the list tags. provide a description and a code snipet. 
Why / when would you want to use list tags over the default option.

File: packages/storybook-html/src/components/accordion/accordion.docs.mdx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    <div class="accordion__content" role="region" aria-hidden="true" id="content-111" aria-labelledby="accordion-title-111">
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum auctor eros dui, vitae iaculis leo ornare eget. Etiam tristique lobortis ligula non mattis. Ut dapibus libero neque, vel tincidunt nunc tempus eget.</p>
+    </div>
+    </div>
+    ...
+    ...
+</div>
+```
+
+### b. Secondary
```
Comment (Bcrunk) [inline packages/storybook-html/src/components/accordion/accordion.docs.mdx:75]: CAn we rename this... well at least provide some context. Make it clear this is a smaller version of the accordions

**Review by Bcrunk — COMMENTED** [review]
You have accessibility tags used but you don;t mention them in an accessibility section or when to use them... should not be a replacement for full accessibility documentation, just some notes to make it clear.

File: packages/storybook-html/src/components/card/card.docs.mdx [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+<StatusBadge.Stable />
+
+<a href="https://github.com/uq-its-ss/design-system/issues" class="uq-button uq-button--text" target="_blank">Check for open issues</a>
+
+## Overview
+
+The card component is used to provide a brief summary of content or a task, often with a link to more detail. Cards are frequently displayed alongside other cards to group related content or tasks.
+
+## Default example
```
Comment (Bcrunk) [inline packages/storybook-html/src/components/card/card.docs.mdx:14]: I think your first example could be a group of three.. just to show the common usage.

File: packages/storybook-html/src/components/card/card.docs.mdx [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+  <Story id="components-cards--card-default-with-link" />
+</Canvas>
+
+```html
+
+```
+
+## Variant examples
+
+### a. Card square overlay title
```
Comment (Bcrunk) [inline packages/storybook-html/src/components/card/card.docs.mdx:38]: I think you need to adjust the naming... these are 

Overlay cards 

Overlay card, Square

File: packages/storybook-html/src/components/card/card.docs.mdx [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+
+<Canvas withSource="none">
+  <Story id="components-cards--card-square-overlay-title" />
+</Canvas>
+
+```html
+
+```
+
+### b. Card square overlay text
```
Comment (Bcrunk) [inline packages/storybook-html/src/components/card/card.docs.mdx:48]: Overlay card, with sub title 

Overlay card, with secondary text

**Review by Bcrunk — COMMENTED** [review]
Code Examples

File: packages/storybook-html/src/components/card/card.docs.mdx (hunk 3 above)
Comment (uqtgeor4) [reply packages/storybook-html/src/components/card/card.docs.mdx:14]: Done

File: packages/storybook-html/src/components/card/card.docs.mdx (hunk 5 above)
Comment (uqtgeor4) [reply packages/storybook-html/src/components/card/card.docs.mdx:48]: Done

File: packages/storybook-html/src/components/accordion/accordion.docs.mdx (hunk 2 above)
Comment (uqtgeor4) [reply packages/storybook-html/src/components/accordion/accordion.docs.mdx:75]: Done

File: packages/storybook-html/src/components/accordion/accordion.docs.mdx (hunk 1 above)
Comment (uqtgeor4) [reply packages/storybook-html/src/components/accordion/accordion.docs.mdx:35]: Done

File: packages/storybook-html/src/components/card/card.docs.mdx (hunk 4 above)
Comment (uqtgeor4) [reply packages/storybook-html/src/components/card/card.docs.mdx]: Done

File: packages/storybook-html/src/components/card/card.docs.mdx [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+    </a>
+    <div class="card__content">
+      <h2 class="card__title">
+        <a class="card__link" href="#">Title of item</a>
+      </h2>
+      <div class="card__date" tabindex="0">Sunday 16 August, 2015</div>
+      <div class="card__location">
+        <a class="card__link" href="#">Gatton campus</a>
+      </div>
+      <div class="card__summary" tabindex="0">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod
```
Comment (Bcrunk) [inline packages/storybook-html/src/components/card/card.docs.mdx:37]: I am not sure we need tabindex in these examples.... tabindex should be avoided at alll times as the nature of UQ sites means we do not control all content going in and therefore can not manage the whole page tabindex, better to let the content flow naturally.

**Review by Bcrunk — COMMENTED** [review]
We are missing a number of examples now from cards. 

Lets discuss some of the formatting a bit more.

**Comment by Bcrunk** [comment]
Changes 

- one default example (move all variants to the Variant section) 
- Include "card--compact" variant
- Remove "Tabindex="0" 
- Simplify the dafualt example, remove date and location
- Create new variant called "Card with additional text" , show date and location text options. 
- Card with Link, remove date and location. 
- Overlay cards, remove repetitive title text, should be "Overlay card with additional text" (remove "square") repeat for others. 
- Remove unneeded class names, drupal class names, these are not needed. 
- Also simplfy overlay card examples, remove content that is not relevant to the example. 
- Group like examples .... Square - Portrait - Landscape ... then follow with other variations.

File: packages/storybook-html/src/components/card/card.docs.mdx (hunk 6 above)
Comment (uqtgeor4) [reply packages/storybook-html/src/components/card/card.docs.mdx:37]: Done

**Review by Bcrunk — APPROVED** [review]

**Pull request merged**